src/Unirest/Request.php

```diff
@@ -420,7 +419,7 @@ public static function getFormattedHeaders($headers)
         $combinedHeaders = array_change_key_case(array_merge((array) $headers, self::$defaultHeaders));
 
         foreach ($combinedHeaders as $key => $val) {
-            $formattedHeaders[] = self::getHeaderString($key, $val);
+            $formattedHeaders[] = $val;
```

Comment:
this is wrong, cURL needs the headers array to be in the format: `key: value` if this is not working for you, then its for a different reason (check your server code)